memberships/models.py

```diff
@@ -4,10 +4,14 @@
 from django.db import transaction
 from django.conf import settings
 from django.utils import timezone
-
 from .services import StripeGateway
 
 
+COUNTRY_CHOICES = ["countries"]
```

Comment:
This is not being used so it can be removed.

Reply:
Fair cop - future improvement